### ISSUE TITLE
Any plan to provide training log?

### ISSUE THREAD

**Issue by zehuichen123** [JeffWang987/OpenOccupancy#16]
Hi, thx for your great work. Is there any plan to provide the training log of the model?

**Comment by JeffWang987** [comment]
Yes, the training logs will be released with the pre-trained models (at this month).

**Comment by JeffWang987** [comment]
Already uploaded, check [here](https://github.com/JeffWang987/OpenOccupancy/blob/main/docs/assets.md).

**Comment by pmj110119** [comment]
It seems that there's something wrong with log files?  The loss is always 1.0 or 5.0, no change.

**Comment by JimPlayboy** [comment]
Hi, the baidu cloud link for the logs of lidar-based baseline seems to be cancelled. Could you please fix this link?
[assets.md](https://github.com/JeffWang987/OpenOccupancy/blob/main/docs/assets.md) 
LiDAR-based baseline [link](https://pan.baidu.com/s/1byc-eBwyrNIOd5okm6CNZQ) (code:vrub)

**Comment by JeffWang987** [comment]
> Hi, the baidu cloud link for the logs of lidar-based baseline seems to be cancelled. Could you please fix this link? [assets.md](https://github.com/JeffWang987/OpenOccupancy/blob/main/docs/assets.md) LiDAR-based baseline [link](https://pan.baidu.com/s/1byc-eBwyrNIOd5okm6CNZQ) (code:vrub)

Already fixed
